ViAn/GUI/mainwindow.cpp

```diff
@@ -281,26 +281,26 @@ void MainWindow::on_slider_click(int new_pos, QPoint local_mouse_pos){
  * @return The current position of the mouse pointer on the slider
  */
 int MainWindow::slider_pos_under_mouse(QPoint local_mouse_pos) {
-    float pos_ratio = local_mouse_pos.x() / (float )ui->videoSlider->size().width();
-    int slider_range = ui->videoSlider->maximum() - ui->videoSlider->minimum();
-    return ui->videoSlider->minimum() + slider_range * pos_ratio;
+    float pos_ratio = local_mouse_pos.x() / (float )ui->video_slider->size().width();
+    int slider_range = ui->video_slider->maximum() - ui->video_slider->minimum();
+    return ui->video_slider->minimum() + slider_range * pos_ratio;
 }
 
 /**
- * @brief MainWindow::on_videoSlider_valueChanged
+ * @brief MainWindow::on_video_slider_valueChanged
  * Gets called when the value of the slider is changed
  * Moves the slider marker and sends the new frame number to the video player
  * @param newPos current position of the slider
```

Comment:
Change in the comment as well. It might not be from this change but it still should be fixed.